### PR TITLE
modify the image_gc's log to structured logs

### PR DESCRIPTION
**Pull request by timyinshi**
Signed-off-by: timyinshi <shiguangyin@inspur.com>

**What type of PR is this?**
/kind feature

**What this PR does / why we need it**:
modify the image_gc's log to structured logs.
the document is:
https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md
```release-note
None
```

### CODE REVIEW

File: pkg/kubelet/images/image_gc_manager.go [hunk 1]
```diff
@@ -356,23 +356,23 @@ func (im *realImageGCManager) freeSpace(bytesToFree int64, freeTime time.Time) (
 	var deletionErrors []error
 	spaceFreed := int64(0)
 	for _, image := range images {
-		klog.V(5).Infof("Evaluating image ID %s for possible garbage collection", image.id)
+		klog.V(5).InfoS("Evaluating image for possible garbage collection", "imageID", image.id)
 		// Images that are currently in used were given a newer lastUsed.
 		if image.lastUsed.Equal(freeTime) || image.lastUsed.After(freeTime) {
-			klog.V(5).Infof("Image ID %s has lastUsed=%v which is >= freeTime=%v, not eligible for garbage collection", image.id, image.lastUsed, freeTime)
+                      klog.V(5).InfoS("Image's lastUsed >= freeTime, not eligible for garbage collection", "imageID", image.id, "imageLastUsed", image.lastUsed, "freeTime", freeTime)
```
Comment (kikisdeliveryservice) [inline pkg/kubelet/images/image_gc_manager.go:362]: maybe: "lastUsed is >= freeTime"

File: pkg/kubelet/images/image_gc_manager.go [hunk 2]
```diff
@@ -301,7 +301,7 @@ func (im *realImageGCManager) GarbageCollect() error {
 	usagePercent := 100 - int(available*100/capacity)
 	if usagePercent >= im.policy.HighThresholdPercent {
 		amountToFree := capacity*int64(100-im.policy.LowThresholdPercent)/100 - available
-		klog.Infof("[imageGCManager]: Disk usage on image filesystem is at %d%% which is over the high threshold (%d%%). Trying to free %d bytes down to the low threshold (%d%%).", usagePercent, im.policy.HighThresholdPercent, amountToFree, im.policy.LowThresholdPercent)
+		klog.InfoS("[imageGCManager]: Disk usage on image filesystem is over the high threshold,Trying to down to the low threshold.", "usagePercent(%)", usagePercent, "highThreshold(%)", im.policy.HighThresholdPercent, "amountToFree(bytes)", amountToFree, "lowThreshold(%)", im.policy.LowThresholdPercent)
```
Comment (kikisdeliveryservice) [inline pkg/kubelet/images/image_gc_manager.go:304]: "Trying to free bytes down to..."

File: pkg/kubelet/images/image_gc_manager.go (hunk 1 above)
Comment (timyinshi) [reply pkg/kubelet/images/image_gc_manager.go]: ok,thanks

File: pkg/kubelet/images/image_gc_manager.go (hunk 2 above)
Comment (timyinshi) [reply pkg/kubelet/images/image_gc_manager.go]: ok,thanks

**Comment by ehashman** [comment]
Please ensure tests are passing and you have updated your release note.

**Comment by ehashman** [comment]
@timyinshi will you have a chance to update this before code freeze?

**Comment by ehashman** [comment]
/close

superseded by #100196

**Pull request closed without merging**